src/include/ci/driver/kernel_compat.h

```diff
@@ -403,4 +403,9 @@ oo_remap_vmalloc_range_partial(struct vm_area_struct *vma, unsigned long uaddr,
 #define pde_data PDE_DATA
 #endif /* ! EFRM_HAVE_LOWCASE_PDE_DATA */
 
+#ifdef EFRM_HAVE_NETIF_RX_NI
+/* linux < 5.18 */
+#define netif_rx netif_rx_ni
```

Comment:
I'm mildly worried about this, since the symbol `netif_rx` is used in other ways in Linux. On balance I think I'll allow it since I can't come up with a simple case where it'll silently fail, but I wanted to highlight that the risk is there.

Reply:
would be cleaner with new name for the macro.